README.md

```diff
@@ -105,11 +105,19 @@ alias ucsdet_open = ucsdet_open_69;                             // usable alias
 
 # Support status
 
-|           | module |                     |
-|:---------:|:------:|:-------------------:|
-| ✅️       | ucsdet | Charset Detection   |
-| ✅️       | ucnv   | Charset conversion  |
-| ✖        | other  |                     |
+|           | module     |                         |
+|:---------:|------------|-------------------------|
```

Comment:
note: I removed center alignment for second and third column (they are left aligned now)